package.json

```diff
@@ -22,6 +22,7 @@
     "release": "npm test && git commit -am $npm_package_version && git tag $npm_package_version && git push origin master && git push --tags && npm publish"
   },
   "devDependencies": {
+    "@types/node": "^10.12.0",
```

Comment:
@Ovyerus I suppose this imports types for globals like `process`? Just curious, isn't this available out of the box?

Reply:
Yeah that's for `process` because VSCode said it didn't exist. I swear it did exist out of the box a while ago but I can't remember.

Reply:
@Ovyerus Intriguing 🤔